docs/workflow.md

```diff
@@ -0,0 +1,372 @@
+       ![](./images/instance-encounter.png)
+    1. If your callback URL test service was set up correctly, you should receive a notification from the mediator.
+
+## Resources
+
+The following [FHIR Resources](https://www.hl7.org/fhir/resource.html) are used to implement the flow above:
+
+- [Patient](https://www.hl7.org/fhir/patient.html)
+- [Encounter](https://build.fhir.org/encounter.html)
+- [Subscription](https://build.fhir.org/subscription.html)
+- [Organization](https://build.fhir.org/organization.html)
+- [Endpoint](https://build.fhir.org/endpoint.html)
+
+The payload samples provided in the documentation contain placeholder values that you must replace with the actual content. To do so, replace the entire '${}' placeholder with the appropriate value. Be aware that some placeholder keys have the format '_\_IDENTIFIER' and refer to the value found in the 'Resource.identifier[0].value' field. These keys are different from the '_\_ID' placeholders used in the request, which refer to the 'Resource.id' field. It is important to make this distinction as using the wrong value may cause unexpected behavior in the system. Therefore, always ensure that you use the right value in the right context to avoid errors.
+
+**Note:** The payload only contains the required fields or a subset of the possible options. To view all the available fields, please refer to the appropriate FHIR resource specifications.
+
+### `ServiceRequest` Resource
+
+The FHIR `ServiceRequest` resource represents a request for a healthcare service to be performed, such as a diagnostic test or a treatment. It contains information about the requested service, including the type of service, the patient for whom the service is requested, the date/time the service is requested, and the healthcare provider or organization making the request. In the context of the LTFU workflow, we are using this resource to request a CHW follow-up in the CHT.
+
+#### `POST ${MEDIATOR_ENDPOINT}/service-request`
+
+This endpoint triggers the creation of a `record` on `CHT` and a `Subscription` resource on FHIR. The endpoint associated with the `Organization` resource in the requester is used as the callback URL for the `Subscription` which gets called when FHIR receives an `Encounter` resource with matching `Patient` identifier. The callback endpoint receives a FHIR `Subscription` response as its payload whenever the request is fulfilled. To learn more about FHIR
+subscriptions, you can visit the official documentation [here](https://build.fhir.org/subscription.html).
+
+```http
+POST ${OPENHIM_ENDPOINT}/service-request
+
+{
+    "intent": "order",
+    "subject": {
+        "reference": "Patient/${PATIENT_IDENTIFIER}"
+    },
+    "requester": {
+        "reference": "Organization/${ORGANIZATION_IDENTIFIER}"
+    }
+    ,
+    "status": "active"
+}
+```
+
```

Comment:
Please mention that this second `json` object is an example of a response body. Same for all 4 resources below.